Podcast/UserSettings.swift

```diff
@@ -25,6 +25,11 @@ class UserSettings: NSObject {
         super.init()
         loadData()
     }
+
+    // reset shared instance upon logout
+    static func reset() {
+        UserSettings.sharedSettings = UserSettings()
```

Comment:
Also probably don't want to reset their entire local settings upon logout. that would piss me off greatly

Reply:
You are right. I changed this to not reset.